ayon_api/utils.py

```diff
@@ -89,6 +89,17 @@ def get_default_settings_variant() -> str:
     return os.environ.get(DEFAULT_VARIANT_ENV_KEY) or "production"
 
 
+def get_machine_name() -> str:
+    """Get machine name.
+
+    Returns:
+        str: Machine name.
+
+    """
+    return platform.node()
```

Comment:
Uh... oopsie?

Reply:
For future reference, fixed with https://github.com/ynput/ayon-python-api/pull/252